demos/common/cpp_gapi/utils_gapi/include/utils_gapi/backend_description.hpp

```diff
@@ -26,8 +26,8 @@ struct BackendDescription {
 
 struct BackendsConfig: ModelConfig {
     BackendsConfig(const ModelConfig &src,
-                   const std::string &mean_values = "",
-                   const std::string &scale_values = "");
+                   const std::vector<float> &mean_values = "",
```

Comment:
Looks `vector<float>` cannot be initialized by `string`

Reply:
Oops, careless error!